src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -145,12 +145,16 @@ async function getAdditionalFacilityInfo(futureAppointments) {
   return facilityData;
 }
 
-export function fetchFutureAppointments() {
+export function fetchFutureAppointments({ includeRequests = true } = {}) {
```

Comment:
The only appointments list page uses this action creator, as well as the canceled page, so we have to make this optional.

Reply:
This syntax looks interesting. Do we need to set a default param to `{}` if we're only grabbing `includeRequests`?

Reply:
I think without `= {}` calls like `fetchFutureAppointments()` (ones without an params) will fail. It works that way in Node, at least.